Makefile

```diff
@@ -84,6 +95,24 @@ test: reports
     -o log_cli=true -o log_level=INFO -o log_file=reports/tests.log \
     tests/* 2>&1 | tee reports/tests.log'
 
+.PHONY: test-ui
+test-ui: reports
+	$(PW_DEBUG) $(PIPENV) run pytest \
+	-v \
+	--junitxml=reports/junit_ui.xml \
+	--html=./reports/pytest_ui.html \
+	-o log_cli=true \
+	-o log_level=$(LOG_LEVEL) \
+	-o log_file=reports/tests_ui.log \
+	tests/ui/test.py 2>&1 | tee reports/tests_ui.log
+
+.PHONY: test-ui-headed
+test-ui-headed: 
+	$(PW_DEBUG) $(PIPENV) run pytest \
+	-vvs \
+	--headed \
+	tests/ui/test.py 2>&1 | tee reports/tests.log
```

Comment:
Can you add 
```
		--video=on \
		--screenshot=on \
		--slowmo=100 \
```
(or what is required to view the tests while they are running) to the local test?